### ISSUE TITLE
[feature request] send GPS location 

### ISSUE THREAD

**Issue by hermann-san** [guardianproject/ChatSecureAndroid#585]
It would be great if ChatSecure would be able to send the current phone location (from GPS) via a Google maps url and/or a small maps image with the location.

**Comment by n8fr8** [comment]
yes, it would! we are working on this for a future release. you can share a location from Google Maps into ChatSecure currently.
